src/main/webapp/app/shared/info-panel/info-panel.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+    selector: 'jhi-info-panel',
+    templateUrl: './info-panel.component.html',
+    styleUrls: ['./info-panel.scss'],
+})
+export class InfoPanelComponent implements OnInit {
+    @Input() panelHeader: string;
+    @Input() panelDescriptionHeader: string;
+
+    constructor() {}
+
+    /**
+     * Do nothing on initialization.
+     */
+    ngOnInit() {}
```

Comment:
If we don't need this, then we shouldn't implement `OnInit`.

Reply:
Done